plugins/utils/guess-indent.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
```

```suggestion
}:
with lib;
helpers.neovim-plugin.mkNeovimPlugin config {
```

Even though this is not recommended in nixpkgs, we adopted this convention across the nixvim codebase.

---

Should we reconsider this? Perhaps we could `inherit (lib) X Y Z` instead?

At least in new files - I'm not proposing we go around changing this in existing code for the sake of it.

---

+1 on *not* using `with`

`nixpkgs` has a tendency on getting rid of it, at least partially (NixOS/nixpkgs#292468) and [best practices] also suggests that `inherit` should be preferred.

[best practices]: https://nix.dev/guides/best-practices#with-scopes